### PR TITLE
New design 2022 - for preview

### PR DESCRIPTION
**Pull request by postescu**

### CODE REVIEW

File: src/components/header/header.tsx [hunk 1]
```diff
@@ -188,14 +197,25 @@ function Header() {
                                         display: {
                                           xs: 'none',
                                           sm: 'inline-block',
+                                          whiteSpace: 'nowrap',
```
Comment (NickLanam) [inline src/components/header/header.tsx:200]: Did you mean to shift this down one line?

File: src/theme.ts [hunk 2]
```diff
@@ -204,6 +205,22 @@ const themeOptions: ThemeOptions = {
...
+        root: {
+          padding: '0 8px',
+          margin: 0,
+        },
+      },
+    },
+    MuiListItemText: {
+      styleOverrides: {
+        root: {
+          padding: '0 8px',
```
Comment (NickLanam) [inline src/theme.ts:219]: This is affecting _all_ list items on the page, including `<ul>`s in blog post content. Probably meant to use a more local change.

Comment (postescu) [reply src/theme.ts]: fixed

**Review by NickLanam — APPROVED** [review]
This is in a great state now! Happy to make this our new base point.

**Review by aimichelle — APPROVED** [review]

**Review by aimichelle — APPROVED** [review]

**Pull request merged**